### DISCUSSION TITLE
Unable to login on  server after install and reboot

### DISCUSSION THREAD

**Discussion by x011** [angristan/wireguard-install#420] [Help]
After running the script to install and generate a couple of users, the server lost connection with the internet and is not longer accessible via ssh or any other service after rebooting.
This happened twice on centos7 and ubuntu20 vps instances.
Any idea of what's causing this?

**Comment by angristan** [comment]
Maybe something is wrong with the network configuration and interfaces fail to get up? Do you have access to a web console with your VPS provider? If so, you can check the log output when the VPS boots and see if there is something failing.

**Comment by ZheGuangZeng** [comment]
Same Issue
